zookeeper-api/src/main/java/org/apache/helix/zookeeper/impl/client/FederatedZkClient.java

```diff
@@ -479,8 +479,18 @@ public long getCreationTime(String path) {
 
   @Override
   public List<OpResult> multi(Iterable<Op> ops) {
-    throwUnsupportedOperationException();
-    return null;
+    if (ops == null) {
+      throw new NullPointerException("ops must not be null.");
+    }
+    String op_path = ops.iterator().next().getPath();
```

Comment:
Two things:
Let's unify the variable naming with camelcase. 
Can we embed the zk path check in the for-loop? There is a small overhead of instantiating this extra iterator.